data/net/minecraft/nbt/CompoundTag.mapping

```diff
@@ -1,109 +1,99 @@
 	METHOD getBoolean (Ljava/lang/String;)Z
-		COMMENT Retrieves a boolean value using the specified key, or false if no such key was stored. This uses the getByte method.
 		ARG 1 key
 	METHOD getByte (Ljava/lang/String;)B
-		COMMENT Retrieves a byte value using the specified key, or 0 if no such key was stored.
 		ARG 1 key
 	METHOD getByteArray (Ljava/lang/String;)[B
-		COMMENT Retrieves a byte array using the specified key, or a zero-length array if no such key was stored.
 		ARG 1 key
 	METHOD getCompound (Ljava/lang/String;)Lnet/minecraft/nbt/CompoundTag;
-		COMMENT Retrieves a NBTTagCompound subtag matching the specified key, or a new empty NBTTagCompound if no such key was stored.
 		ARG 1 key
 	METHOD getDouble (Ljava/lang/String;)D
-		COMMENT Retrieves a double value using the specified key, or 0 if no such key was stored.
 		ARG 1 key
 	METHOD getFloat (Ljava/lang/String;)F
-		COMMENT Retrieves a float value using the specified key, or 0 if no such key was stored.
 		ARG 1 key
 	METHOD getId ()B
-		COMMENT Gets the type byte for the tag.
 	METHOD getInt (Ljava/lang/String;)I
-		COMMENT Retrieves an integer value using the specified key, or 0 if no such key was stored.
 		ARG 1 key
 	METHOD getIntArray (Ljava/lang/String;)[I
-		COMMENT Retrieves an int array using the specified key, or a zero-length array if no such key was stored.
 		ARG 1 key
 	METHOD getList (Ljava/lang/String;I)Lnet/minecraft/nbt/ListTag;
-		COMMENT Gets the NBTTagList object with the given name.
 		ARG 1 key
-		ARG 2 type
+		ARG 2 tagType
 	METHOD getLong (Ljava/lang/String;)J
-		COMMENT Retrieves a long value using the specified key, or 0 if no such key was stored.
 		ARG 1 key
 	METHOD getLongArray (Ljava/lang/String;)[J
 		ARG 1 key
 	METHOD getShort (Ljava/lang/String;)S
-		COMMENT Retrieves a short value using the specified key, or 0 if no such key was stored.
 		ARG 1 key
 	METHOD getString (Ljava/lang/String;)Ljava/lang/String;
-		COMMENT Retrieves a string value using the specified key, or an empty string if no such key was stored.
 		ARG 1 key
 	METHOD getTagType (Ljava/lang/String;)B
-		COMMENT Gets the ID byte for the given tag key
+		COMMENT Gets the byte identifier of the tag of the specified {@code key},
+		COMMENT or {@code 0} if no tag exists for the {@code key}.
 		ARG 1 key
 	METHOD getUUID (Ljava/lang/String;)Ljava/util/UUID;
 		ARG 1 key
 	METHOD hasUUID (Ljava/lang/String;)Z
 		ARG 1 key
+	METHOD lambda$static$0 (Lcom/mojang/serialization/Dynamic;)Lcom/mojang/serialization/DataResult;
+		ARG 0 dynamic
+	METHOD lambda$static$1 (Lnet/minecraft/nbt/CompoundTag;)Lcom/mojang/serialization/Dynamic;
+		ARG 0 compoundTag
 	METHOD merge (Lnet/minecraft/nbt/CompoundTag;)Lnet/minecraft/nbt/CompoundTag;
-		COMMENT Deep copies all the tags of {@code other} into this tag, then returns itself.
+		COMMENT Copies all the tags of {@code other} into this tag, then returns itself.
+		COMMENT @see #copy()
```

Comment:
Another standards point worth clarifying: this is a reference to it's own class, so it does not need to be fully qualified. Is this correct?

- IF the goal is to provide cross-version migration ability of javadoc links, then this is not useful
- IF the goal is to make javadoc links work regardless of the presence of a particular import statement, then this is fine.

@ standards people (oh look it's me) thoughts?

Reply:
That would be my understanding. The reason I use this method is that how copying works is applied based on the description within the javadoced method. The goal here I would say is to make the link work without an import statement.